packages/backend-common/src/database/postgres.ts

```diff
@@ -54,10 +69,35 @@ export function buildPgDatabaseConfig(
 }
 
 /**
- * Gets the postgres connection config
+ * Builds a knex Postgres database connection for database creation
+ *
+ * @param dbConfig The database config
+ */
+function buildPgDatabaseAdminConfig(dbConfig: JsonValue) {
```

Comment:
Really no need to have this and `buildPgPluginConfig` as separate functions right? Gets pretty verbose